shared/android/app/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+# For more information about using CMake with Android Studio, read the
```

Comment:
This file builds our JSI native code. This is what we will register to interop between JS and our binding.cpp file